analyzers/tests/SonarAnalyzer.UnitTest/TestCases/VariableUnused.cs

```diff
@@ -32,9 +32,17 @@ void F1()
 
             b(5);
 
+            Action notUsed = () => // Noncompliant
+            {
+                Console.WriteLine("Hello world.");
+            };
+
             string c;
             c = "Hello, world!";
             Console.WriteLine(c);
+            
+            string f; // FN
+            f = "Hello, world!";
```

Comment:
Why is this an FN? This is a useless write operation, but there should be another rule handling these cases.

Reply:
Isn't this considered also an unused variable?
IMO it's similar to `string f = "Hello"`.

Reply:
But I don't see the test case `string f = "Hello";` either.

Reply:
it's not FN, it's handled by the DeadStores rule